### DISCUSSION TITLE
SETTINGS_HEADER_TABLE_SIZE

### DISCUSSION THREAD

**Discussion by khteh** [nodejs/help#5040] [Q&A]
### Affected URL(s)

https://nodejs.org/api/http2.html

### Description of the problem

Does `node:http2` honour `SETTINGS_HEADER_TABLE_SIZE` setting?

**Comment by bnoordhuis** [comment]
Yes, it's the `headerTableSize` option. Converting to a discussion.

**Reply by khteh** [reply]
I use spdy for HTTP2 but it is doesn't hornour the setting from AWS ALB which doesn't support dynamic header compression
